app/src/main/java/com/example/sharingang/items/ItemDao.kt

```diff
@@ -19,4 +19,7 @@ interface ItemDao {
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     fun insert(items: List<Item>)
-}
\ No newline at end of file
+
+    @Query("DELETE FROM item")
```

Comment:
No need to specify the item id?

Reply:
Here we want to clear the table entirely
It fixes the issue where the items are deleted on Firebase but stay in the cache.

Reply:
Oh yeah I see, makes sense.